[33mWARNING [0m conda_build.config:config.py:322 Setting build arch. This is only useful when pretending to be on another arch, such as for rendering necessary dependencies on a non-native arch. I trust that you know what you're doing.
[32mINFO    [0m conda_build.variants:variants.py:235 Adding in variants from /home/runner/work/conda-build/conda-build/tests/test-recipes/split-packages/_line_up_python_compiled_libs/conda_build_config.yaml
[32mINFO    [0m conda_build.metadata:metadata.py:854 Attempting to finalize metadata for libxyz
[35mDEBUG   [0m conda_index.index:__init__.py:680 found subdirs {'noarch', 'win-64'}
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:102 CondaIndexCache channel_root=/home/runner/work/_temp/parallel/popen-gw0/test_python_line_up_with_compi0, subdir=noarch db_filename=/home/runner/work/_temp/parallel/popen-gw0/test_python_line_up_with_compi0/noarch/.cache/cache.db cache_is_brand_new=True
[32mINFO    [0m conda_index.index.convert_cache:convert_cache.py:150 Migrate database
[32mINFO    [0m conda_index.index.convert_cache:convert_cache.py:199 CONVERT .cache
[32mINFO    [0m conda_index.index.convert_cache:convert_cache.py:150 Migrate database
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:431 noarch listdir
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:450 noarch save fs state
[35mDEBUG   [0m conda_index.index:__init__.py:759 noarch find packages to extract
[35mDEBUG   [0m conda_index.index:__init__.py:771 noarch extract 0 packages
[32mINFO    [0m conda_index.index:__init__.py:806 noarch cached 0 B from 0 packages at 0 B/second
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:102 CondaIndexCache channel_root=/home/runner/work/_temp/parallel/popen-gw0/test_python_line_up_with_compi0, subdir=win-64 db_filename=/home/runner/work/_temp/parallel/popen-gw0/test_python_line_up_with_compi0/win-64/.cache/cache.db cache_is_brand_new=True
[32mINFO    [0m conda_index.index.convert_cache:convert_cache.py:150 Migrate database
[32mINFO    [0m conda_index.index.convert_cache:convert_cache.py:199 CONVERT .cache
[32mINFO    [0m conda_index.index.convert_cache:convert_cache.py:150 Migrate database
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:431 win-64 listdir
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:450 win-64 save fs state
[35mDEBUG   [0m conda_index.index:__init__.py:759 win-64 find packages to extract
[35mDEBUG   [0m conda_index.index:__init__.py:771 win-64 extract 0 packages
[32mINFO    [0m conda_index.index:__init__.py:806 win-64 cached 0 B from 0 packages at 0 B/second
[32mINFO    [0m conda_index.index:__init__.py:578 Completed noarch
[32mINFO    [0m conda_index.index:__init__.py:578 Completed win-64
[35mDEBUG   [0m conda_index.index:__init__.py:680 found subdirs {'noarch', 'win-64'}
[32mINFO    [0m conda_index.index:__init__.py:653 Channeldata subdir: noarch
[35mDEBUG   [0m conda_index.index:__init__.py:654 noarch read repodata
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:102 CondaIndexCache channel_root=/home/runner/work/_temp/parallel/popen-gw0/test_python_line_up_with_compi0, subdir=noarch db_filename=/home/runner/work/_temp/parallel/popen-gw0/test_python_line_up_with_compi0/noarch/.cache/cache.db cache_is_brand_new=False
[35mDEBUG   [0m conda_index.index:__init__.py:662 noarch channeldata finished
[32mINFO    [0m conda_index.index:__init__.py:653 Channeldata subdir: win-64
[35mDEBUG   [0m conda_index.index:__init__.py:654 win-64 read repodata
[35mDEBUG   [0m conda_index.index.sqlitecache:sqlitecache.py:102 CondaIndexCache channel_root=/home/runner/work/_temp/parallel/popen-gw0/test_python_line_up_with_compi0, subdir=win-64 db_filename=/home/runner/work/_temp/parallel/popen-gw0/test_python_line_up_with_compi0/win-64/.cache/cache.db cache_is_brand_new=False
[35mDEBUG   [0m conda_index.index:__init__.py:662 win-64 channeldata finished
[35mDEBUG   [0m conda_index.index:__init__.py:1155 _maybe_write /home/runner/work/_temp/parallel/popen-gw0/test_python_line_up_with_compi0/index.html to /home/runner/work/_temp/parallel/popen-gw0/test_python_line_up_with_compi0/index.html
[35mDEBUG   [0m conda_index.index:__init__.py:670 write channeldata
[35mDEBUG   [0m conda_index.index:__init__.py:1155 _maybe_write /home/runner/work/_temp/parallel/popen-gw0/test_python_line_up_with_compi0/channeldata.json to /home/runner/work/_temp/parallel/popen-gw0/test_python_line_up_with_compi0/channeldata.json
[35mDEBUG   [0m urllib3.util.retry:retry.py:594 Incremented Retry for (url='/pkgs/main/win-64/repodata.json'): Retry(total=2, connect=None, read=None, redirect=None, status=None)
[33mWARNING [0m urllib3.connectionpool:connectionpool.py:813 Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='repo.anaconda.com', port=443): Read timed out. (read timeout=60.0)")': /pkgs/main/win-64/repodata.json
[32mINFO    [0m conda_build.metadata:metadata.py:854 Attempting to finalize metadata for py-xyz
[32mINFO    [0m conda_build.metadata:metadata.py:854 Attempting to finalize metadata for xyz